src/mol-theme/color/residue-name.ts

```diff
@@ -71,7 +122,8 @@ export const ResidueNameColorThemeParams = {
     lightness: PD.Numeric(1, { min: -6, max: 6, step: 0.1 }),
     colors: PD.MappedStatic('default', {
         'default': PD.EmptyGroup(),
-        'custom': PD.Group(getColorMapParams(ResidueNameColors))
+        'custom': PD.Group(getColorMapParams(ResidueNameColors)),
+        'residue-charge': PD.Group(getColorMapParams(ChargedResidueColors))
```

Comment:
This feels quite misplaced to me -- I would prefer this was a new color theme altogether. Basically a copy-paste of this theme, but rename it to `residue-charge` and perhaps allow alternative color schemes.

What do you think?

Reply:
I agree, ideally putting it directly below/above Hydrophobicity

Reply:
Yes, I didn't want to duplicate the code too much but it makes sense

> I agree, ideally putting it directly below/above Hydrophobicity

Now the options are listed alphabetically so I will keep it that way